web/src/actions/editActivity/activityList.js

```diff
@@ -1,6 +1,16 @@
-import { ADD_APD_ITEM, EDIT_APD, REMOVE_APD_ITEM } from '../editApd/symbols';
+import {
+  ADD_APD_ITEM,
+  APD_ACTIVITIES_CHANGE,
+  EDIT_APD,
+  REMOVE_APD_ITEM
+} from '../editApd/symbols';
 import { updateBudget } from '../budget';
 
+// extract activites from state, safely
+const getActivites = (getState) => (
+  (((getState() || {}).apd || {}).data || {}).activities || []
+)
+
```

Comment:
Navigating the object this way may not be necessary. I was getting errors from calls to `getState` within a few tests.